### PR TITLE
fix opening tabs with transitional

### PR DESCRIPTION
**Pull request by brindy**
Task/Issue URL: https://app.asana.com/0/414235014887631/1206769708570992/f
Tech Design URL:
CC:

**Description**:
Pages that open new tabs in a certain way will get stuck on `about:blank`

**Steps to test this PR**:
1. Visit https://finance.yahoo.com/news/ocean-biomedical-inc-announces-publication-131500736.html?guccounter=1 and get to the article content
2. Click on a link in the article. 
3. A new tab should open and navigate to the content
4. Visit https://chrisbrind.rocks and click on one of the Connect buttons (instagram, tiktok, mastodon) - should all open in a new tab and load successfully (I know this website is awful sorry)

### CODE REVIEW

File: DuckDuckGo/MainViewController.swift [hunk 1]
```diff
@@ -1833,7 +1833,8 @@ extension MainViewController: TabDelegate {
         showBars()
         currentTab?.dismiss()
 
-        let newTab = tabManager.addURLRequest(navigationAction.request,
+        // Don't use a request or else the page gets stuck on "about:blank"
+        let newTab = tabManager.addURLRequest(nil,
```
Comment (bwaresiak) [inline DuckDuckGo/MainViewController.swift:1837]: What if we do want to navigate to given URL through request, e.g. on `window.open` calls?

**Comment by federicocappelli** [comment]
I tested your scenario and all works, the code seems ok but I'm worried about our (supposed) lack of UI tests for this very important part of the browser, how do we guarantee that all other scenarios are still working?

File: DuckDuckGo/MainViewController.swift (hunk 1 above)
Comment (bwaresiak) [reply DuckDuckGo/MainViewController.swift:1837]: Ok, I've checked the docs: "The web view returned must be created with the specified configuration. WebKit loads the request in the returned web view." so this should be fine.

**Comment by brindy** [comment]
@federicocappelli I don't disagree, but at the same time this isn't the place for that.  The existence of UI tests at all is relatively new and don't have the bandwidth to create them for this quick fix.  Can you create a project task for that, please?

**Review by federicocappelli — APPROVED** [review]
LGTM

**Pull request merged**